authorization-dotnet/BouncyCastleCryptography/BouncyCastleCryptography/BouncyCastleCryptography.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="BouncyCastleCryptographyTests\**" />
+    <EmbeddedResource Remove="BouncyCastleCryptographyTests\**" />
+    <None Remove="BouncyCastleCryptographyTests\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BouncyCastle.Cryptography" Version="2.3.0" />
```

Comment:
In VS it is giving a warning as this package version contains a vulnerability, we can upgrade it to latest version 2.3.1